vtysh/vtysh.c

```diff
@@ -3474,6 +3474,158 @@ DEFPY (show_route_map,
 	return CMD_SUCCESS;
 }
 
+static void show_prefix_list_send(afi_t afi, const char *prefix_list,
+				  const char *seq, enum display_type dtype,
+				  bool json)
+{
+	unsigned int i;
+	bool first = true;
+	char command_line[128];
+
+	if (afi == AFI_IP)
+		snprintf(command_line, sizeof(command_line), "show ip prefix-list ");
```

Comment:
It MUST be `do show ip ...`, otherwise it won't work if you enter `do show ip prefix-list ...`.